wisp/build.gradle.kts

```diff
@@ -1,178 +1,12 @@
-import org.gradle.api.tasks.testing.logging.TestExceptionFormat
-import org.jetbrains.kotlin.gradle.plugin.KotlinPluginWrapper
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
-plugins {
-  alias(libs.plugins.kotlinGradlePlugin) apply false
-  alias(libs.plugins.kotlinBinaryCompatibilityPlugin) apply false
-  alias(libs.plugins.protobufGradlePlugin) apply false
-  alias(libs.plugins.mavenPublishGradlePlugin) apply false
-  alias(libs.plugins.versionsGradlePlugin)
-  alias(libs.plugins.versionCatalogUpdateGradlePlugin)
-  alias(libs.plugins.dependencyAnalysisPlugin)
-}
-
-dependencyAnalysis {
-  issues {
-    all {
-      onAny {
-        severity("fail")
-      }
-    }
-    project(":wisp-logging-testing") {
-      onUnusedDependencies {
-        // False positive.
-        exclude(":wisp-logging")
-      }
-    }
-  }
-}
-
-repositories {
-  mavenCentral()
-  gradlePluginPortal()
-}
-
-buildscript {
-  repositories {
-    mavenCentral()
-    gradlePluginPortal()
-  }
-}
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinJvm
 
 subprojects {
-  buildscript {
-    repositories {
-      mavenCentral()
-      gradlePluginPortal()
-    }
-  }
-
-  repositories {
-    mavenCentral()
-    maven(url = "https://s3-us-west-2.amazonaws.com/dynamodb-local/release")
-  }
 
-  if (!path.startsWith(":wisp-bom")) {
+  if (!path.startsWith(":wisp:wisp-bom")) {
```

Comment:
Is this file necessary? I've usually seen the subdir not have a build file.

Reply:
For now, yes, because unlike Misk which adds the plugins in each submodule's `build.gradle`, wisp defines them in top level build.gradle. This minimizes the changes for this merge and makes it easier to debug and test the migration changes for wisp modules as well as look at previous changes on the same file. But in the future, we can cleanup Misk and just apply common plugins in the main build.file.